packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -49,6 +49,9 @@ const ALERT_LAST_DETECTED = `${ALERT_NAMESPACE}.last_detected` as const;
 // kibana.alert.reason - human readable reason that this alert is active
 const ALERT_REASON = `${ALERT_NAMESPACE}.reason` as const;
 
+// kibana.alert.url - url which will take the user directly to a view of the alert
+const ALERT_URL = `${ALERT_NAMESPACE}.url` as const;
```

Comment:
Could you add `ALERT_URL` to an alert schema for 8.8, similar to how we updated the BaseFields for 8.4 (x-pack/plugins/security_solution/common/detection_engine/schemas/alerts/8.4.0/index.ts) when adding `ALERT_RULE_INDICES`?

Reply:
I am doing it another PR.